modules/tasks/tasks/src/test/java/org/motechproject/tasks/service/impl/TaskActionExecutorTest.java

```diff
@@ -257,5 +326,9 @@ private boolean serviceMethodInvoked() {
         public void serviceMethod() {
             invoked = true;
         }
+
+        public void serviceMethod(String string) {
```

Comment:
This method, with string param, is invoked by reflection in 'shouldExecuteTaskAndUsePostActionParameters' test.